### ISSUE TITLE
New metric: IsInFeed

### ISSUE THREAD

**Issue by realhagie** [Steem-FOSSbot/steem-fossbot-voter#28]
how about a metric for all posts which appear in my feed ?

Regards

**Comment by thrize** [comment]
Good idea, this is definitely possible with the steem js library so can do. Thanks!

We'd need to only look at the newest posts in the feed too, so as an internal implementation note, we'd need to keep track of that.

**Comment by thrize** [comment]
**Accepted this suggestion, will be implemented**

Metric name will be ```post_is_in_feed```
